src/types/guilds/membership_screening_field_types.ts

```diff
@@ -0,0 +1,4 @@
+export enum MembershipScreeningFieldTypes {
```

Comment:
Missing "doc resource" link here.

Reply:
Does not exist as of now. They have removed the doc. But it still exists, so should we keep the type?

Reply:
Remove, it is then